src/extension.ts

```diff
@@ -191,6 +191,16 @@ export function activate(context: vscode.ExtensionContext): void {
         })
     );
 
+    // uploadJSFromTS
+    context.subscriptions.push(
+        vscode.commands.registerCommand('extension.vscode-janus-debug.uploadJSFromTS', (param) => {
+            if (vscode.window.activeTextEditor) {
+                const _param = vscode.window.activeTextEditor.document;
+                commands.uploadJSFromTS(loginData, _param);
+            }
```

Comment:
Having names beginning with an underline (e.g. `_param` or `_uploadScript`)  is not very idiomatic TS and is only used, if at all, for class properties AFAIK. You should use keywords/mechanisms like `public`, `private`, `export` to distinguish between API and non-API or to hide implementation details. What is the reason for those underlines?